docs/configs/pdf/conf.py

```diff
@@ -332,6 +332,7 @@
 
 rst_prolog = """
 .. _installer: https://github.com/digital-asset/daml/releases/download/v{release}/daml-sdk-{release}-windows.exe
+.. _Artifactory: https://digitalasset.jfrog.io/ui/repos/tree/General/sdk-ee
```

Comment:
this was broken before but we didn’t error out on sphinx warnings in the pdf build.